src/docs/MLOS/MLOSUserGuide.adoc

```diff
@@ -17,7 +17,7 @@ include::references/Glossary.adoc[]
 
 == Get Started
 
-To submit your first Machine Learning workflow to *ProActive Scheduler*, link:../admin/ProActiveAdminGuide.html#_run_the_proactive_scheduler[install] it in
+To submit your first ML workflow to *ProActive Scheduler*, link:../admin/ProActiveAdminGuide.html#_run_the_proactive_scheduler[install] it in
```

Comment:
You should introduce the abreviation before using it for the first time. 
ML ==> Machine Learning (ML)